api/types/networking.go

```diff
@@ -385,6 +391,26 @@ func (c *ClusterNetworkingConfigV2) SetCaseInsensitiveRouting(cir bool) {
 	c.Spec.CaseInsensitiveRouting = cir
 }
 
+// GetSSHDialTimeout returns the timeout to be used for SSH connections.
+// If the value is not set, or was intentionally set to an infinite value,
```

Comment:
```suggestion
// If the value is not set, or was intentionally set to a negative value,
```

Not sure how you'd deliberately set it to an infinite value.